be/src/formats/parquet/group_reader.cpp

```diff
@@ -219,7 +219,8 @@ StatusOr<size_t> GroupReader::_read_range_round_by_round(const Range<uint64_t>&
                                                          ChunkPtr* chunk) {
     const std::vector<int>& read_order = _column_read_order_ctx->get_column_read_order();
     size_t round_cost = 0;
-    DeferOp defer([&]() { _column_read_order_ctx->update_ctx(round_cost); });
+    double first_selectivity = -1;
+    DeferOp defer([&]() { _column_read_order_ctx->update_ctx(round_cost, first_selectivity); });
```

Comment:
Will this miss the calculation when returning early on LOC:240

Reply:
it's by design, early return, means hit_count is zero, min_round will deal this kind scenario.